test/arbitrum/GMDStrategy.js

```diff
@@ -450,10 +455,10 @@ describe("GMDStrategy", function () {
 
         await vault
             .connect(deployer)
-            ["updateStrategyDebtRatio(address,uint256)"](
-                strategy.address,
-                5000
-            );
+        ["updateStrategyDebtRatio(address,uint256)"](
+            strategy.address,
+            5000
+        );
         mine(1);
```

Comment:
Advice: do not use mining directly, utilize hardhat network helpers.

Reply:
to be refactored next time